src/scss/_fun-facts.scss

```diff
@@ -1,127 +1,152 @@
 //---------------Break Points---------------//
 $bp-facts-small: 50em; //800px
 $bp-facts-medium: 56.5em; //904px
-$bp-facts-large: 81.25em; //1300px
+$bp-facts-large: 64em; //1024px
+$bp-facts-xlarge: 81.25em; //1300px
 
 .fun-fact__heading {
-	font-family: $font-avenir-medium;
-	font-size: 1.25em; //20px
-	font-weight: 700;
-	color: $color-brown;
-	text-align: left;
-	margin-bottom: 0.25em; //4px
+  font-family: $font-avenir-medium;
+  font-size: 1.25em; //20px
+  font-weight: 700;
+  color: $color-brown;
+  text-align: left;
+  margin-bottom: 0.25em; //4px
 }
 
 .fun-fact__text {
-	font-family: $font-avenir-roman;
-	font-size: 1em; //16px
-	font-weight: 400;
-	color: $color-brown;
-	text-align: left;
+  font-family: $font-avenir-roman;
+  font-size: 1em; //16px
+  font-weight: 400;
+  color: $color-brown;
+  text-align: left;
 }
 
 //-----Duck-----//
 .icon-duck {
-	background-position: 175% 100%;
-	background-size: 10em auto; //160px
-	position: relative;
-
-	@media(min-width: $bp-facts-small) {
-		background-position: 100% 100%;
-		cursor: pointer;
-
-		&:hover .fun-fact--duck{
-			display: inline-block;
-		}
-
-		&:active .fun-fact--duck {
-			display: inline-block;
-		}
-	}
+  background-position: 205% 100%;
+  background-size: 10em auto; //160px
+  position: relative;
+
+  @media(min-width: $bp-facts-small) {
+    background-position: 100% 100%;
+    cursor: pointer;
+  }
+  
+  @media(min-width: $bp-facts-small) and (max-width: 1023px) {
```

Comment:
It looks like you use variables in most other places, is there ar reason for the very specific px value here?